src/backend/expungeservice/models/charge_types/misdemeanor_class_bc.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from expungeservice.models.charge import ChargeType
+from expungeservice.models.charge import ChargeUtil
+from expungeservice.models.expungement_result import TypeEligibility, EligibilityStatus
+
+
+@dataclass(frozen=True)
+class MisdemeanorClassBC(ChargeType):
+    type_name: str = "Misdemeanor Class B or C"
+    expungement_rules: str = """Convictions for misdemeanors are generally eligible under ORS 137.225(5)(b).
```

Comment:
Did you check if "ORS 137.225(5)(b)" is the right category still?

Reply:
I hadn't checked, but yes this is unchanged.